.github/labels.md

```diff
@@ -0,0 +1,238 @@
+# Issue and Pull Request labels
+
+This document describes the intended usage of GitHub Issue and Pull Request
+labels within the FreeOrion project.  In this document when referring to
+__Issues__ GitHub Issues are meant and when referring to __PR__ GitHub
+Pull Requests are meant.
+
+
+## Purpose of labels
+
+Labels allow attaching common search and filter terms to Issues/PRs.  This makes
+it easier for developers, managers and reporters to track the purpose, scope and
+state of Issues/PRs without explicitly reading through lengthy comments, code
+reviews or commits.  Please follow this guide as closely as possible to avoid
+duplicating work and wasting time of other developers and to enable better
+documentation on decisions made.
+
+
+## Label categories
+
+Labels are grouped in categories.  Those categories group together a common
+concept like the resolving state of an Issue/PR or the target platform where the
```

Comment:
> resolving ->revolving, optionally current

`Revolving` sounds odd; using 'current processing state' now.

> What is meant by target platform here? I read this as relating to operating systems, which are not covered by existing labels.

I was meant as a general example and yes, I had the OS in mind.  But yes, I should use category we really have.